catalog/env.template

```diff
@@ -78,6 +78,11 @@ AIRFLOW_CONN_DATA_REFRESH=http://ingestion_server:8001
 # server.
 AIRFLOW_CONN_STAGING_DATA_REFRESH=http://ingestion_server:8001
 
+# Connect to sensitive terms list used for filtered index creation in the data refresh.
+# For local development, the mock terms list is served by the catalog indexer worker. Change to the
+# full production URL in order to use the production sensitive terms list.
+AIRFLOW_CONN_SENSITIVE_TERMS="http://catalog_indexer_worker:8003/static/mock_sensitive_terms.txt"
```

Comment:
It's not required, since we can set this through the Admin UI (as we have with several other connections/variables). We could make an infrastructure change to set it as an environment variable though; actually, I'm not clear how we've drawn the line between which variables should be configured where 🤔 What do you think?

Reply:
Using the Admin UI sounds good, I believe that's in line with decisions we've made in the past (https://github.com/WordPress/openverse-infrastructure/issues/215). I forgot that environment variables aren't necessary for configuring these. I realised that I didn't personally have a clear picture of environment variables vs Airflow variables/connections, and that some of our code muddies that distinction... and presents problems for advancements in our Airflow configuration, management, and execution. I've opened #4863 to address those inconsistencies.

I've started [a discussion here](https://github.com/WordPress/openverse/discussions/4865) to work towards an approach more in line with the rest of our applications (using infrastructure as code).

Reply:
I think this is fine for now! One disadvantage of declaring this in the Admin UI is that the password (if needed) for the services isn't codified in the same way in our infrastructure repo. Noting that for now, I'll add more thorough thoughts to that discussion when I get a chance to look at that discussion.